src/Zigzag_persistence/example/example_usage_filtered_zigzag_persistence_with_storage.cpp

```diff
@@ -0,0 +1,55 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hannah Schreiber
+ *
+ *    Copyright (C) 2024 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#include <iostream>
+
+#include <gudhi/filtered_zigzag_persistence.h>
+
+using Zigzag_persistence = Gudhi::zigzag_persistence::Filtered_zigzag_persistence_with_storage<>;
+
+int main(int argc, char* const argv[]) {
```

Comment:
```suggestion
int main() {
```
since argc and argv are unused